sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed get link from session check it is disposed or not. ([#25718](https://github.com/Azure/azure-sdk-for-java/issues/25718))
```

Comment:
Fixed "what".

"get link from session check it is disposed or not" is not a bug that you've fixed here, I assume.

Also do you really need to use unreleased lib here?

Reply:
I see. Fixed client may get disposed links from session.

Servicebus SDK do not need to use the latest azure-core package.

Reply:
There is a problem, `azure-core-amqp` always use the latest `azure-core`,  I use old-version `azure-core` and latest `azure-core-amqp` in servicebus, and the module-info.java set `requires transitive com.azure.core.amqp`  , the IDE hint module servicebus doesn't read azure-core, it won't build seccessfully in IDE ( command line build successfully). I think we should keep use the latest version of `azure-core`, I have no idea why we set transitive and still put `azure-core` in pom file.

Reply:
pom is Maven (for package management, also include dependency)
module-info is Java (for encapsulation and modularity, also include dependency)

That is 2 different thing, just both involve dependency.